using/self_service.md

```diff
@@ -322,3 +322,118 @@ Once allocation has been created, you can see its access information:
 2. Account ID that needs to be passed to *sbatch* command when scheduling a job.
 
 ![Details of an allocation](../images/project-batch-details.png)
+
+## Adding a public SSH key to a profile
+1. Generate your SSH keypair (on Windows: using PuTTYgen, Linux: using OpenSSH)
+2. Click Details on the left side of the screen
+3. Select SSH keys
+4. Click Add SSH key
+5. Paste there contents of a public SSH key
+
+![Adding public SSH key](../images/adding-ssh-key.png)
+
+## Python management service
+
+### Prerequisites
+
+1. Your project in ETAIS has a Private Cloud. Please refer to [Adding a VPC](#adding-a-vpc)
+2. Your account has your SSH public key. Please refer to [Adding public SSH key](#adding-a-public-ssh-key-to-a-profile)
+3. Virtual machine is create and properly configured. For details please refer to [Adding a VM](#adding-a-vm)
+   1. Make sure image is either Debian 9 or Ubuntu 16.04
+   2. Select your previously uploaded SSH public key in the form
+   3. Ssh and Web security groups should be assigned 
+   4. "Auto-assign a floating IP" option in Networks combobox should be selected (or any existing IP address)
+4. Wait for VM to start.
+
+>If VM status is active, it does not mean that a Linux distribution has been installed, you need to wait a few moments before you will be allowed to connect to a VM.
```

Comment:
Installed? What did you have in mind?

Reply:
Somehow I can't connect to a VM right away when its status turns ACTIVE

Reply:
Ok, but distro installation is not related. Booting a VM might take a bit of time for SSHD to get up. Please rephrase, e.g. "If you cannot connect to a newly created VM at once, wait for 4-5 minutes to make sure that OS has booted and SSH daemon has been started."